cobalt/black_box_tests/black_box_tests.py

```diff
@@ -228,10 +275,20 @@ def Run(self):
     if self.proxy_port == '-1':
       return 1
 
+    run_cobalt_tests = True
+    run_evergreen_tests = False
     if (f'{_launcher_params.platform}/{_launcher_params.config}'
         in _DISABLED_BLACKBOXTEST_CONFIGS):
-      logging.warning('Blackbox tests disabled for platform:%s config:%s',
-                      _launcher_params.platform, _launcher_params.config)
+      run_cobalt_tests = False
+      logging.warning(
+          'Cobalt blackbox tests disabled for platform:%s config:%s',
+          _launcher_params.platform, _launcher_params.config)
+
+    if (f'{_launcher_params.platform}/{_launcher_params.config}'
```

Comment:
Readability nit: this got duplicated, pull it into a separate var: `launching_config=f'{_launcher_params.platform}/{_launcher_params.config}'`

Reply:
Good call, done.